internal/integration/harness.go

```diff
@@ -395,6 +408,35 @@ func run(t *testing.T, actionsOrOptions ...ActionOrOption) {
 				})
 			}
 
+			if opts.resetPubSub {
+				Infof("Resetting pubsub")
+				envars := []string{"COMPOSE_IGNORE_ORPHANS=True"}
+				err = exec.CommandWithEnv(ctx, log.Debug, rootDir, envars, "docker", "compose", "-f", "internal/dev/docker-compose.redpanda.yml", "-p", "ftl", "up", "-d", "--wait").RunBuffered(ctx)
+				assert.NoError(t, err)
+
+				client, err := sarama.NewClient(redPandaBrokers, sarama.NewConfig())
+				assert.NoError(t, err)
+				defer client.Close()
+
+				clusterAdmin, err := sarama.NewClusterAdminFromClient(client)
+				assert.NoError(t, err)
+				defer clusterAdmin.Close()
+
+				// Try to reset up to 3 times
+				// There can be a slight delay for consumer groups to be empty after a previous run of FTL ends
+				for i := range 15 {
```

Comment:
This PR does not close kafka consumers cleanly so it can take about 10s for pubsub for consumer groups to be empty. Resetting pubsub has to wait for this to happen first.

Reply:
Maybe removed the `volumes` section from docker compose so the data does not persist over restarts?